tachyon/zk/plonk/halo2/proof_reader.h

```diff
@@ -56,7 +56,7 @@ class ProofReader {
     proof_.advices_commitments_vec.resize(num_circuits_);
     size_t num_advice_columns = constraint_system.num_advice_columns();
     for (size_t i = 0; i < num_circuits_; ++i) {
-      proof_.advices_commitments_vec[i].resize(num_advice_columns);
+      proof_.advices_commitments_vec[i].resize(num_advice_columns, C::Zero());
```

Comment:
How can it be compiled..? I think it should accept `std::vector<C>`!

```suggestion
      proof_.advices_commitments_vec[i].resize(num_advice_columns);
```

Reply:
`advices_commitments_vec` is vector of vector so `advices_commitments_vec[i]` is vector of `F`.